### PR TITLE
DOCSUP-6702: document the timezoneOffset function

### PR DESCRIPTION
**Pull request by lehasm**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #19962

### CODE REVIEW

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -57,6 +57,44 @@ int32samoa: 1546300800
...
+timezoneOffset(datetime)
+```
+
+**Arguments**
+
+-   `datetime` — date and time. [DateTime](../../sql-reference/data-types/datetime.md) or [DateTime64](../../sql-reference/data-types/datetime64.md). 
+
+**Returned value**
+
+-   offset from UTC in seconds.
```
Comment (adevyatova) [inline docs/en/sql-reference/functions/date-time-functions.md:77]:
```suggestion
-   Offset from UTC in seconds. 
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 2]
```diff
@@ -57,6 +57,44 @@ int32samoa: 1546300800
...
+
+**Syntax**
+
+``` sql
+timezoneOffset(datetime)
+```
+
+**Arguments**
+
+-   `datetime` — date and time. [DateTime](../../sql-reference/data-types/datetime.md) or [DateTime64](../../sql-reference/data-types/datetime64.md).
```
Comment (adevyatova) [inline docs/en/sql-reference/functions/date-time-functions.md:73]:
```suggestion
-   `datetime` — Date and time. [DateTime](../../sql-reference/data-types/datetime.md) or [DateTime64](../../sql-reference/data-types/datetime64.md). 
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 3]
```diff
@@ -57,6 +57,44 @@ int32samoa: 1546300800
 
 `toTimeZone(time_utc, 'Asia/Yekaterinburg')` changes the `DateTime('UTC')` type to `DateTime('Asia/Yekaterinburg')`. The value (Unixtimestamp) 1546300800 stays the same, but the string representation (the result of the toString() function) changes from `time_utc:   2019-01-01 00:00:00` to `time_yekat: 2019-01-01 05:00:00`.
 
+## timezoneOffset {#timezoneoffset}
+
+Returns a timezone offset from [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time) in seconds. The function takes into account [Daylight saving time](https://en.wikipedia.org/wiki/Daylight_saving_time) and historical timezone changes at the specified date and time.
```
Comment (adevyatova) [inline docs/en/sql-reference/functions/date-time-functions.md:62]:
```suggestion
Returns a timezone offset in seconds from [UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time). The function takes into account [daylight saving time](https://en.wikipedia.org/wiki/Daylight_saving_time) and historical timezone changes at the specified date and time.
```

**Comment by lehasm** [comment]
@KochetovNicolai please have a look.

By the way, some other functions have aliases with different letter case: `time[zZ]one` It seems that there is no alias for this function so far. If it is intended I can try to add it.

**Comment by KochetovNicolai** [comment]
> By the way, some other functions have aliases with different letter case: time[zZ]one It seems that there is no alias for this function so far. If it is intended I can try to add it.

Yes, it should be useful. 
I think it is enough to call 
`factory.registerAlias("timeZoneOffset", "timeZoneOffset");` here:
https://github.com/ClickHouse/ClickHouse/blob/master/src/Functions/timezoneOffset.cpp#L14

**Review by KochetovNicolai — APPROVED** [review]

**Comment by lehasm** [comment]
@KochetovNicolai
> factory.registerAlias("timeZoneOffset", "timeZoneOffset");
Just to make it clear. Does we need small letter `z` in the second argument? Like
`factory.registerAlias("timeZoneOffset", "timezoneOffset");`

**Review by KochetovNicolai — APPROVED** [review]

**Pull request merged**